plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java

```diff
@@ -165,9 +165,9 @@ public void testAnalyzeWithSchemaEvolution()
                         VALUES
                           ('nationkey', null, 25, 0, null, '0', '24'),
                           ('regionkey', null, 5, 0, null, '0', '4'),
-                          ('name', 1314.0, 25, 0, null, null, null),
-                          ('info', 4417.0, 25, 0, null, null, null),
-                          (null, null, null, null, 25, null, null)""");
+                          ('name', 1907.9999999999998, 25, 0, null, null, null),
+                          ('info', 4417.0, 25, 0.1, null, null, null),
+                          (null, null, null, null, 50, null, null)""");
```

Comment:
add a comment why 50

i am surprised `testAnalyzeWithSchemaEvolution` is the only method that shows a user-visible behavioral regression (stats inaccuracy after whole file deleted), especially that it has nothing to with schema evolution.  Please add simpler dedicated test that shows that.